controllers/discovery/searchDomain.js

```diff
@@ -35,7 +36,9 @@ const SearchDomain = async (req, res) => {
         ) AS "user_id_follower"
       FROM "domain_page" AS "Domain" 
       LEFT JOIN "user_follow_domain" AS "domainFollower" ON "Domain"."domain_page_id" = "domainFollower"."domain_id_followed" 
-      WHERE "Domain"."domain_name" ILIKE :likeQuery 
+      WHERE 
+        "Domain"."domain_name" ILIKE :likeQuery 
+        AND "Domain"."credder_score" >= ${CREDDER_MIN_SCORE}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While the code is correct, it's not a good practice to directly inject variables into SQL queries as it can lead to SQL injection attacks. It would be better to use parameterized queries.
```diff
- AND "Domain"."credder_score" >= ${CREDDER_MIN_SCORE}
+ AND "Domain"."credder_score" >= :credderMinScore
```
And then add `credderMinScore: CREDDER_MIN_SCORE` to the replacements object in the query.



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
this is defined as constant, so will not lead to SQL Injection